java/arcs/core/data/proto/manifest.proto

```diff
@@ -106,6 +106,7 @@ message TypeProto {
     TypeVariableProto variable = 6;
     SingletonTypeProto singleton = 7;
     CountTypeProto count = 8;
+    ListTypeProto list = 9;
```

Comment:
Are this and other changes in this proto file handled on typescript side?

Reply:
Good point, I'll move this to a new PR now

Reply:
Done in https://github.com/PolymerLabs/arcs/pull/5715